### ISSUE TITLE
Scrollbar looks awkward in OS X 10.7+

### ISSUE THREAD

**Issue by macgyver** [ptrm/prettysimple-adiummsgstyle#2]
The scrollbar in this theme was slick and streamlined in Snow Leopard and earlier, but Lion changed the UI paradigm by making scrollbars that did not displace content in the view, or inherently take up any space at all.  Is there anyway we can get a version of this that is more consistent with the Lion UI, which uses the standard OS scrollbars and no longer has the custom scrollbar?

**Comment by macgyver** [comment]
D'oh, just realized that "No scrollbars" is already an option as one of the Message Style's **Variants**, thanks again for the great theme!

**Comment by ptrm** [comment]
Yes, it's an option, but since Lion has nice looking scrollbars as a default, it would be good indeed to get rid of the custom ones on 10.7+, as you mentioned. I don't presently have a Mac at hand unfortunately, but it should not be a big trouble to simply add a Variant with default scrollbars, I'll try to come up with something soon.

**Comment by macgyver** [comment]
That would be rad, I noticed that scrolling doesn't seem to work in the No Scrollbar variant so I'm looking forward to this!

**Comment by ptrm** [comment]
Sorry. That was meant to be a week I suppose. However, good news is I've got my Mac working again after months, so the fix is more sure to come now.

(the close wasn't meant)
